cmake/OpenCVDumpPreprocessor.cmake

```diff
@@ -0,0 +1,35 @@
+# Utility
+function(dump_gcc_preprocessor fname exe extra_flags)
+  set(src "${CMAKE_BINARY_DIR}/preprocessor/empty")
```

Comment:
`CMAKE_CURRENT_BINARY_DIR` here and below

Reply:
Done.